api/agent/config.go

```diff
@@ -102,6 +107,10 @@ func NewAgentConfig() (*AgentConfig, error) {
 		cfg.EnableNBResourceTracker = true
 	}
 
+	if _, ok := os.LookupEnv(EnvEnableReadOnlyRootFs); ok {
```

Comment:
Does this imply that we need a change in the pure runner to enable this feature?

Reply:
Yes, runner env would need an update.